File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int, int> countMap;
        for (auto i : nums) {
            countMap[i]++;
            if(countMap[i] >= 2) {
                return true;
            }
        }

        return false;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        d = dict()
        for index, number in enumerate (nums) :
            if number in d :
                d[number] += 1
                if d[number] == 2:
                    return True
            else : 
                d[number] = 1
        return False
        # print(d)